### ISSUE TITLE
Enhancement - Update labels instead of removing all / adding new ones 

### ISSUE THREAD

**Issue by rdelgatte** [Decathlon/pull-request-labeler-action#6]
Scenario: 

1. Opening a new Pull request with documentation and tests files
  - Before: `No Labels`
  - After: `["documentation", "tests"]`
2. Adding a new label (manually - which is not configured): `["documentation", "tests", "manual-label"]`
3. Pushing once more but removing the documentation files from the pull request changes, I should have: `["tests", "manual-label"]`

So I should only remove `["documentation"]` in this case
